install-k6.js

```diff
@@ -0,0 +1,91 @@
+const { exec } = require('child_process')
+const { existsSync } = require('fs')
+
+const K6_VERSION = 'v0.55.0'
+const K6_PATH_MAC_AMD = `k6-${K6_VERSION}-macos-amd64`
+const K6_PATH_MAC_ARM = `k6-${K6_VERSION}-macos-arm64`
+const K6_PATH_WIN_AMD = `k6-${K6_VERSION}-windows-amd64`
+
+const existsK6 = (os, arch) => {
+  return existsSync(`resources/${os}/${arch}/k6`)
+}
+
+const getMacOSK6Binary = () => {
+  const command = `
+# download binaries
+curl -LO https://github.com/grafana/k6/releases/download/${K6_VERSION}/${K6_PATH_MAC_AMD}.zip
+curl -LO https://github.com/grafana/k6/releases/download/${K6_VERSION}/${K6_PATH_MAC_ARM}.zip
+
+# unzip & smoke test
+unzip ${K6_PATH_MAC_AMD}.zip
+unzip ${K6_PATH_MAC_ARM}.zip
+${K6_PATH_MAC_AMD}/k6 version
+${K6_PATH_MAC_ARM}/k6 version
+
+# move to resource folder
+mv ${K6_PATH_MAC_AMD}/k6 resources/mac/x86_64
+mv ${K6_PATH_MAC_ARM}/k6 resources/mac/arm64
+
+# cleanup
+rm ${K6_PATH_MAC_AMD}.zip
+rm ${K6_PATH_MAC_ARM}.zip
+rmdir ${K6_PATH_MAC_AMD}
+rmdir ${K6_PATH_MAC_ARM}
+`
+
+  executeCommand(command, {})
+}
+
+const getWindowsK6Binary = () => {
+  const command = `
+# download binaries
+Invoke-WebRequest -Uri "https://github.com/grafana/k6/releases/download/${K6_VERSION}/${K6_PATH_WIN_AMD}.zip" -OutFile "k6-windows-amd64.zip"
+
+# unzip & smoke test
+Expand-Archive -Path "k6-windows-amd64.zip" -DestinationPath "."
+${K6_PATH_WIN_AMD}\\k6.exe version
+
+# move to resource folder
+Move-Item -Path "${K6_PATH_WIN_AMD}\\k6.exe" -Destination resources\\win\\x86_64
+`
```

Comment:
I noticed the temp files were still there after the download was complete (they were also showing up in `git status`). We can add a clean up function here similar to what we have for macOS


```suggestion
# clean up
del k6-windows-amd64.zip
Remove-Item -Recurse -Force ${K6_PATH_WIN_AMD}
`
```